dist/components/product/_product.dust

```diff
@@ -0,0 +1,38 @@
+{#product}
+<div class="c-product {?isHighlight}c--highlight{/isHighlight}" role="group">
+    <a class="c-product__title" href="{url}" title="{title}" rel="external">
+        <h2>{title}</h2>
+    </a>
+
+    {#image}
+        <div class="c-product__image">
+            <a href="{url}" title="{title}" rel="external">
```

Comment:
Is there a reason `rel="external"` is being declared? I imagine most sites' products will be internal.